CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ### Fixed
 
+- Fixed a bug where it was no longer possible to edit meshes or enable view tools after the new shortcut improvements were added to the editor.
```

Comment:
This should be in a new ## [Unreleased] section and not in 5.2.0

Reply:
Oops! Fixed in [cb02fe5](https://github.com/Unity-Technologies/com.unity.probuilder/pull/487/commits/cb02fe5a2792e9639cd384b686adfef47d538d04)